packages/flutter_tools/test/commands.shard/permeable/packages_test.dart

```diff
@@ -37,10 +38,15 @@ void main() {
       final String projectPath = await createProject(tempDir, arguments: arguments);
       final File pubspec = globals.fs.file(globals.fs.path.join(projectPath, 'pubspec.yaml'));
       String content = await pubspec.readAsString();
-      content = content.replaceFirst(
-        '\ndependencies:\n',
-        '\ndependencies:\n  $plugin:\n',
-      );
+      final List<String> contentLines = LineSplitter.split(content).toList();
+      final int depsIndex = contentLines.indexOf('dependencies:');
+      if (depsIndex != -1) {
+        contentLines.replaceRange(depsIndex, depsIndex + 1, <String>[
+          'dependencies:',
+          '  $plugin:',
+        ]);
+        content = contentLines.join('\n');
```

Comment:
I don't know if it matters that we aren't maintaining the line endings, probably not.

Reply:
Yeah, I'm also worried that adding logic to retain Windows line endings would tickle the Cirrus problem.